config/default.js

```diff
@@ -0,0 +1,189 @@
+const {
+  VERSIONS,
+  RESOURCES
+} = require('@asymmetrik/node-fhir-server-core').constants;
+
+// // Set up whitelist
+// let whitelist_env = env.WHITELIST && env.WHITELIST.split(',').map(host => host.trim()) || false;
+//
+// // If no whitelist is present, disable cors
+// // If it's length is 1, set it to a string, so * works
+// // If there are multiple, keep them as an array
+// let whitelist = whitelist_env && whitelist_env.length === 1
+// 	? whitelist_env[0]
+// 	: whitelist_env;
+
+//
+// let supportedResources = [RESOURCES.ACCOUNT,
+//                           RESOURCES.PATIENT,
+//                           RESOURCES.OBSERVATION,
+//                           RESOURCES.MEDICATIONSTATEMENT,
+//                           RESOURCES.ADVERSEEVENT,
+//                           RESOURCES.ALLERGYINTOLERANCE,
+//                           RESOURCES.IMMUNIZATION,
+//                           RESOURCES.MEDICATIONADMINISTRATION,
+//                           RESOURCES.CONDITION,
+//                           RESOURCES.DEVICE,
+//                           RESOURCES.PROCEDURE,
+//                           RESOURCES.PRACTITIONER,
+//                           RESOURCES.ENCOUNTER,
+//                           RESOURCES.MEDICATIONREQUEST];
+// let profiles = {};
+// for (var i in supportedResources){
+//   let profile = supportedResources[i];
+//   profiles[profile] = { service: new PassThroughService(profile),
+//                         versions: [ VERSIONS['3_0_1'] ]};
+// }
+
+
+let fhirClientConfig = {
+  baseUrl: 'http://localhost',
+  debug: true
+};
+
+/**c
+ * @name fhirServerConfig
+ * @summary @asymmetrik/node-fhir-server-core configurations.
+ */
+let fhirServerConfig = {
+  auth: {
+    // This servers URI
+    resourceServer: 'http://localhost:3001',
+    //
+    // if you use this strategy, you need to add the corresponding env vars to docker-compose
+    //
+    strategy: {
+      name: 'bearer',
+      useSession: false,
+      service: './src/strategies/bearer.strategy.js'
+    },
+  },
+  server: {
+    // support various ENV that uses PORT vs SERVER_PORT
+    port: 3001,
+    // allow Access-Control-Allow-Origin
+    corsOptions: {
+      maxAge: 86400,
+      origin: ['*', 'http://mm221745-pc.mitre.org:3000']
```

Comment:
Is this necessarily going to have to be configured by each server? Is there a way to make this implicit somehow?

Reply:
I thought that I had removed the individual entry and that * was the only thing left to allow all hosts.  I'll remove it.